ocaml/xapi-cli-server/cli_operations.ml

```diff
@@ -935,7 +935,8 @@ let gen_cmds rpc session_id =
     ; Client.PIF.(
         mk get_all_records_where get_by_uuid pif_record "pif" []
           [
-            "uuid"; "device"; "VLAN"; "mac"; "network-uuid"; "currently-attached"
+            "uuid"; "device"; "VLAN"; "MAC"; "network-uuid"; "currently-attached";
+            "host-id"
```

Comment:
It's `host-uuid`... and please make sure you test it ;)

Reply:
For some reason, `xe pif-list` still does not output either MAC or host-id here, after integrating this patch into the RPM build chain to be sure not to miss anything.

Reply:
It works for me.